src/components/abstract/QComponent.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
```

Comment:
React is not used. (This was probably copied from a js file that uses jsx like `<Component />`, which requires `React` to be in scope)